Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -282,6 +282,35 @@ class SearchLanguageSetting: Setting {
     }
 }
 
+// Opens the News for language settings
+class NewsLanguageSetting: Setting {
```

Comment:
Can we move it to own file?

Reply:
I will suggest to leave it here because all settings are placed in this file.